src/components/SetupAssistant/SetupAssistantItem.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { ProgressRing } from '../Progress';
+
+const SetupAssistantItem = ({
+  children,
+  isExpanded,
+  title,
+  stepProgress,
+  step,
+  renderProgress,
+  renderExpanded,
+  renderAddon,
+  onExpand
+}, { assetBasePath }) => (
+  <li className="slds-setup-assistant__item">
+    <article className="slds-setup-assistant__step">
+      <div className={cx('slds-summary-detail', { 'slds-is-open': isExpanded })}>
+        {onExpand && (
+          <button
+            type="button"
+            className="slds-button slds-button_icon slds-m-right_x-small slds-m-top_x-small"
+            onClick={onExpand}
+          >
+            <svg className="slds-button__icon slds-summary-detail__action-icon">
+              <use xlinkHref={`${assetBasePath}icons/utility-sprite/svg/symbols.svg#switch`} />
+            </svg>
+          </button>
+        )}
+        <div className="slds-container_fluid">
+          <div className="slds-summary-detail__title">
+            <div className="slds-setup-assistant__step-summary">
+              <div className="slds-media">
```

Comment:
You should be able to use a `<MediaObject />` here if you add a `bodyClassName` prop to it.